DRAFT: Cloud REST, clean intro docs (#830)
grafana/k6-docs: src/data/markdown/docs/06 Cloud REST API/01 Cloud REST API/04 Tests.md

## List tests

Returns all tests within a specified project.

> If no project is specified, k6 returns the default project.

Author: ```suggestion
If you don't specify a project, k6 returns the default project.
```
